app/index.html

```diff
@@ -205,6 +205,36 @@ <h1 class="hero__title">
           </div>
         </div>
       </div>
+
+      <section class="description">
+        <div class="container">
+          <h2 class="description__top-title">Private Jet for Business & Leisure Purposes</h2>

```

Comment:
bad BEM, description__top-title must be inside description__top. Better rename to description__title